tests/test-rest-meta-users-controller.php

```diff
@@ -0,0 +1,106 @@
+<?php
+
+/**
+ * Unit tests covering WP_REST_Users meta functionality.
+ *
+ * @package WordPress
+ * @subpackage JSON API
+ */
+class WP_Test_REST_Meta_Users_Controller extends WP_Test_REST_Controller_Testcase {
+	public function setUp() {
+		parent::setUp();
+
+		$this->user = $this->factory->user->create();
+		wp_set_current_user( $this->user );
+		$this->user_obj = wp_get_current_user();
+		$this->user_obj->add_role( 'adminstrator' );
+		$this->user_obj->add_cap( 'edit_users' );
+		$this->user_obj->add_cap( 'delete_users' );
+	}
+
+	public function test_register_routes() {
+		$routes = $this->server->get_routes();
+
+		$this->assertArrayHasKey( '/wp/v2/users/(?P<parent_id>[\d]+)/meta', $routes );
+		$this->assertCount( 2, $routes['/wp/v2/users/(?P<parent_id>[\d]+)/meta'] );
+		$this->assertArrayHasKey( '/wp/v2/users/(?P<parent_id>[\d]+)/meta/(?P<id>[\d]+)', $routes );
+		$this->assertCount( 3, $routes['/wp/v2/users/(?P<parent_id>[\d]+)/meta/(?P<id>[\d]+)'] );
+	}
+
+	public function test_context_param() {
+		$user_id = $this->factory->user->create();
+		// Collection
+		$request = new WP_REST_Request( 'OPTIONS', '/wp/v2/users/' . $user_id . '/meta' );
+		$response = $this->server->dispatch( $request );
+		$data = $response->get_data();
+		$this->assertEquals( 'edit', $data['endpoints'][0]['args']['context']['default'] );
+		$this->assertEquals( array( 'edit' ), $data['endpoints'][0]['args']['context']['enum'] );
+		// Single
+		$meta_id_basic = add_user_meta( $user_id, 'testkey', 'testvalue' );
+		$request = new WP_REST_Request( 'OPTIONS', '/wp/v2/users/' . $user_id . '/meta/' . $meta_id_basic );
+		$response = $this->server->dispatch( $request );
+		$data = $response->get_data();
+		$this->assertEquals( 'edit', $data['endpoints'][0]['args']['context']['default'] );
+		$this->assertEquals( array( 'edit' ), $data['endpoints'][0]['args']['context']['enum'] );
+	}
+
+	public function test_get_items() {
+		$user_id                    = $this->factory->user->create();
+		$meta_id_serialized         = add_user_meta( $user_id, 'testkey_serialized', array( 'testvalue1', 'testvalue2' ) );
+		$meta_id_serialized_object  = add_user_meta( $user_id, 'testkey_serialized_object', (object) array( 'testvalue' => 'test' ) );
+		$meta_id_serialized_array   = add_user_meta( $user_id, 'testkey_serialized_array', serialize( array( 'testkey1' => 'testvalue1', 'testkey2' => 'testvalue2' ) ) );
+		$meta_id_protected          = add_user_meta( $user_id, '_testkey', 'testvalue' );
+
+		$request = new WP_REST_Request( 'GET', sprintf( '/wp/v2/users/%d/meta', $user_id ) );
+		$response = $this->server->dispatch( $request );
+
+		$this->assertEquals( 200, $response->get_status() );
```

Comment:
This is the line that is failing to pass in travis.  I am not 100% sure why it is failing since it is passing on my local install via VVV.

Failed: https://travis-ci.org/WP-API/wp-api-meta-endpoints/jobs/111276918#L300

@danielbachhuber thoughts?